iidm/iidm-tck/pom.xml

```diff
@@ -54,6 +54,11 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
You also need to add this dependency in the `powsybl-iidm-impl`'s `pom.xml` for the tests to succeed.